package.json

```diff
@@ -18,13 +18,18 @@
     }
   },
   "dependencies": {
+    "@emotion/react": "^11.7.1",
```

Comment:
Oh small thing to note, @emotion/react, @emotion/styled, @mui/icons-material and @mui/material are not needed since we are using MUI v4 and not MUI v5.

Reply:
And can delete package-lock.json since we only using yarn.lock